### PR TITLE
framework/st_things : Modify FOTA protocol.

### PR DESCRIPTION
**Pull request by YiHyunJin**
The server protocol has been modified to accommodate the server.
and remove unused resources.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [518750f0a0e1569548b53a269bbe3575ea27688b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [518750f0a0e1569548b53a269bbe3575ea27688b] - Code Rule Check Result:
framework/src/st_things/things_stack/framework/things_data_manager.c:1979: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('

File: framework/src/st_things/things_stack/framework/things_data_manager.c [hunk 1]
```diff
@@ -282,10 +258,10 @@ static const  struct things_attribute_info_s const gstAttr_oic_r_firmware[] = {
 		.rw = 3
 	},
 	{
-		.key = "description",
+		.key = "supportprotcols",
```
Comment (jarvisjang) [inline framework/src/st_things/things_stack/framework/things_data_manager.c]: 'protcols' looks like a mistype. 'protocols' may be right. :)

**Comment by YiHyunJin** [comment]
Leave a comment

**Comment by seinfra** [comment]
Target : [54a53626f0cae56ae5a4a8c5a71e6df0c3d502de] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [54a53626f0cae56ae5a4a8c5a71e6df0c3d502de] - Code Rule Check Result:
framework/src/st_things/things_stack/framework/things_data_manager.c:1979: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('

**Comment by seinfra** [comment]
Target : [b036a7714884187b748d955d09bc7eb5a9e15dbf] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b036a7714884187b748d955d09bc7eb5a9e15dbf] - Code Rule Check OK.

**Pull request closed without merging**